lld/test/ELF/aarch64-execute-only.s

```diff
@@ -0,0 +1,33 @@
+// REQUIRES: aarch64
+
+// RUN: llvm-mc -filetype=obj -triple=aarch64-linux-none %s -o %t.o
```

Comment:
The triple aarch64-linux-none does not make sense. Just `aarch64`

(While you could use `aarch64-linux`, `aarch64` (unknown ELF) makes it clear that the test doesn't use Linux specific features)